open-source-policy/practice.md

```diff
@@ -1,33 +1,33 @@
 # Practicing our open source policy #
 
-This document is meant to give specific team guidance on putting our [open
-source policy](policy.md) into practice.
+We have an "open-by-default" development policy. This document is meant to
+give specific team guidance on putting our [open source policy](policy.md)
+into practice.
 
-* CISA releases software into the [international public domain](#public-domain).
-* All team members should feel empowered to contribute back to outside open
-  source projects.
+* CISA releases software into the [international public domain](#public-domain)
+* Contribute back to outside open source projects whenever possible.
 * We [develop our software in the open](#working-in-public), while also
-  [protecting sensitive information](#protecting-sensitive-information).
+  [protecting sensitive information](#protecting-sensitive-information)
```

Comment:
Why is the period removed?

Reply:
Resolved in 267f33866baf7df7d71137c503c07968d32d75e2